charts/trusted-attestation-controller/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+name: trusted-attestation-controller
+version: 0.2.0
+appVersion: 1.0
```

Comment:
We keep the version and appVersion the same for now and they need to match the appVersion which the version of the TAC (0.3.0 now but 0.4.0 once new release is done w/ null plugin)